test/uk/gov/hmrc/fileupload/controllers/EnvelopeCheckerSpec.scala

```diff
@@ -38,13 +39,18 @@ class EnvelopeCheckerSpec extends UnitSpec {
   import uk.gov.hmrc.fileupload.ImplicitsSupport.StreamImplicits.materializer
 
   val testRequest = FakeRequest()
+
   val testEnvelopeId = EnvelopeId()
 
+  val defaultFileSize = 10 * 1024 * 1024
+
   "When an envelope is OPEN it" should {
     "be possible to execute an Action" in {
       val expectedAction = Action { req => Ok }
 
-      val wrappedAction = withValidEnvelope(_ => Future(Xor.right("OPEN")))(testEnvelopeId)(expectedAction)
+      val envelopeOpen = Json.parse("""{ "status" : "OPEN", "constraints" : {} }""")
```

Comment:
It'd be nice to add a test case with an envelope that has no constraint attribute at all in addition to empty constraints field.